guacamole-ext/src/main/resources/org/apache/guacamole/protocols/vnc.json

```diff
@@ -19,6 +19,10 @@
         {
             "name"  : "authentication",
             "fields" : [
+                {
+                    "name"  : "username",
+                    "type"  : "TEXT"
```

Comment:
This should actually be `USERNAME`. The two fields are technically implemented identically, but `USERNAME` is defined with username-specific meaning.

See:

https://github.com/apache/guacamole-client/blob/339c920a89ba6d2363bd442d77490b00237ce650/guacamole-ext/src/main/resources/org/apache/guacamole/protocols/rdp.json#L22-L25

Reply:
Ah, yes.  Fixed.